.github/workflows/android.yml

```diff
@@ -57,7 +57,9 @@ jobs:
           path: android/build/nodejs-native-assets
           key: ${{ runner.os }}-nodejs-mobile-${{ hashFiles('src/backend/package-lock.json') }}
       - name: Node
-        uses: actions/setup-node@v1
+        uses: actions/setup-node@v2
+        with:
+          node-version: "12.16.3"
```

Comment:
Interesting I had thought we were already using Node 12 for builds. It's set in `.nvmrc`, but I don't think Github actions pay attention to that file. The other two places this matters are in Bitrise and Cirrus CI. Bitrise respects `.nvmrc` because it uses nvm to manage Node (https://github.com/digidem/mapeo-mobile/blob/develop/bitrise.yml#L326). Cirrus CI uses a Docker container that we build ourselves, and that is currently set to `12.x` e.g. Node v12 latest (https://github.com/digidem/docker-android/blob/master/Dockerfile#L16). So I think we should be ok for this change, just wanted to make sure you are aware of all these pieces.

Reply:
Certain parts of the build process were using Node 12, but they had `node-version` specified. Without it [version 10 was being installed[(https://github.com/digidem/mapeo-mobile/runs/4718848720?check_suite_focus=true#step:6:7) (or used via the cache).

Regardless, the additional context is appreciated 👍